packages/map-template/src/atoms/hasCategoryPropState.js

```diff
@@ -0,0 +1,8 @@
+import { atom } from 'recoil';
+
+const hasCategoryPropState = atom({
```

Comment:
Would it be more clean to have the value from the prop as the value in the atom? We already have atoms for properties (locationId, startZoomLevel etc.), and this boolean state is kinda off pattern-wise.

Reply:
Good idea. Refactored the category atom. ✅